### DISCUSSION TITLE
frigate

### DISCUSSION THREAD

**Discussion by petervk** [tteck/Proxmox#2079] [Ideas]
Is it possible to get a Frigate LXC script?

**Comment by shinyone** [comment]
Here to request the same.    There are lots of threads.  Maybe counting up all of the votes from each thread may push this to the top of the list!  With support for Goggle Coral included.... please.

https://github.com/tteck/Proxmox/discussions/197#discussioncomment-6935150

https://github.com/tteck/Proxmox/discussions/1644

https://github.com/tteck/Proxmox/discussions/1332


LXC required
https://docs.frigate.video/frigate/installation#proxmox

Is there any documentation for https://github.com/tteck/Proxmox/blob/main/misc/frigate-support.sh


From @tteck's ideas list:
@remz1337 wrote:
https://github.com/tteck/Proxmox/discussions/50#discussioncomment-6463382
https://github.com/remz1337/frigate-standalone
<sub>Be cautious and thoroughly evaluate scripts and automation tasks obtained from external sources.</sub>

**Reply by remz1337** [reply]
Happy to share that my standalone script has been running great for a few months now :) I don't have any Coral, I'm using Nvidia GPU with TensorRT, but shouldn't be hard to do.

**Reply by eteixido1** [reply]
Hello @remz1337  and @tteck ,
It would be really nice if you could integrate this script with tteck LXC auto-deploy. I don't know how complicated it is but it would be very useful.

Kind regards and thank you again for your dedication

**Reply by tteck** [reply]
@remz1337 is your script still functional?

**Reply by remz1337** [reply]
@tteck Yes, and my latest revision uses a fixed release of frigate so it shouldn't be an issue.
However this is purely a "it works on my machine" as the script assumes you have an Nvidia GPU passed through, with CUDA already installed.

My suggestion would be that I make the dependency on TensorRT optional (based on some internal flag or script parameter) and so that we can try to get this working with a minimal environment (CPU detection only).

**Reply by eteixido1** [reply]
Hello @tteck are you working in this new script? I think we are a lot of people who are very interested in.

**Reply by timdonovanuk** [reply]
@remz1337 how easy is it to unpick your nvidia dependencies from your script? Is it possible using the frigate UI? i.e. - is using your script a good starting point for getting frigate going, even if you don't intend to use nvidia passthrough? Thanks!

**Reply by remz1337** [reply]
@timdonovanuk @tteck Just pushed an update to my script that defaults to installing without the nvidia dependencies (cpu detector only). Successfully tested on Ubuntu 22 LXC (already tested on Debian 11 too)
